db/migrate/20200908155632_add_total_applications_to_activities.rb

```diff
@@ -0,0 +1,5 @@
+class AddTotalApplicationsToActivities < ActiveRecord::Migration[6.0]
+  def change
+    add_column :activities, :total_applications, :integer
```

Comment:
As this is an integer, perhaps this could default to `0` and not allow NULLs..?

Reply:
Yeah, I wasn't sure about defaults here. Is defaulting to `0` the convention? I guess the reason I didn't default in the end is because I wanted to be sure the user enters a number (by having validations of presence on this step), instead of clicking `continue` by mistake and saving the wrong value. I'm unsure what the best option is in this case. What do you think?

Reply:
Good point there about the forms - if we preset this to `0` does the form get pre-filled with that value? I note further down that `0` has a special (temporary) meaning anyway, so maybe leave this be.

It's probably not a convention, but it's something I often do - possibly because I've been bitten by databases that can't `sum` columns that contain `NULL` in the past 🤷

Reply:
That was my assumption, only because we I defaulted a boolean to `true`, the radio button for the true value gets pre-filled. I thought defaulting to `0` would have the same behaviour (I might be wrong, I'll check). 
Happy to add not allowing `NULL`. Don't want to cause trouble with databases 😅